localization/src/languages/ru.json

```
@@ -1,96 +1,96 @@
-            "DoYouWantChooseDateTime": "Do you want to choose date and time?",
-            "ChooseDateTime": "Choose date and time",
-            "YouHaveChosenTheDateTime": "You have chosen date and time: "
+            "YouHaveChosenTheDate": "Выбранная дата: ",
+            "YouHaveChosenTheTime": "Выбранное время: ",
+            "DoYouWantChooseDateTime": "Хотите выбрать дату и время?",
+            "ChooseDateTime": "Выбор даты и времени",
+            "YouHaveChosenTheDateTime": "Выбранные дата и время: "
         },
         "SigningBox": {
-            "Prompt": "How would you like to sign?",
-            "EnterKey": "Enter Key",
-            "PickSignature": "Pick a signature",
-            "UseSecurityCard": "Use Security Card",
-            "SecurityCards": "Security cards",
-            "Signatures": "Signatures",
-            "AddSignature": "Add signature",
-            "PrivateKey": "Private key",
+            "Prompt": "Как хотите подписать транзакцию?",
+            "EnterKey": "Ввести ключ",
+            "PickSignature": "Выбрать подпись",
+            "UseSecurityCard": "Картой безопасности",
+            "SecurityCards": "Карты безопасности",
+            "Signatures": "Подписи",
+            "AddSignature": "Добавить подпись",
+            "PrivateKey": "Приватный ключ",
             "Sign": "Подписать"
         },
         "Approve": {
-            "Title": "Transaction confirmation",
-            "To": "To",
-            "Recipients": "Recipients",
-            "Total": "Total",
-            "Fees": "Fees",
-            "Signature": "Signature",
-            "Attention": "Attention",
+            "Title": "Подтверждение транзакции",
+            "To": "Кому",
+            "Recipients": "Получатели",
+            "Total": "Всего",
+            "Fees": "Сбор",
+            "Signature": "Подпись",
+            "Attention": "Внимание",
             "AttentionDesc": "This transaction is trying to change the contract and will not be performed",
-            "Confirm": "Confirm",
-            "Cancel": "Cancel"
+            "Confirm": "Подтвердить",
+            "Cancel": "Отмена"
         },
         "TransactionConfirmation": {
-            "Title": "Transaction confirmation",
-            "To": "To",
-            "Recipients": "Recipients",
-            "Total": "Total",
-            "Fees": "Fees",
-            "Signature": "Signature",
-            "Attention": "Attention",
-            "AttentionDesc": "This transaction is trying to change the contract and will not be performed",
-            "Confirm": "Confirm",
-            "Cancel": "Cancel"
+            "Title": "Подтверждение транзакции",
+            "To": "Кому",
+            "Recipients": "Получатели",
+            "Total": "Всего",
+            "Fees": "Сбор",
+            "Signature": "Подпись",
+            "Attention": "Внимание",
+            "AttentionDesc": "Транзакция не будет выполнена, потому что ",
```

```suggestion
            "AttentionDesc": "Транзакция не будет выполнена, потому что пытается изменить контракт",
```
Isn't it more precise?